ecs-cli/modules/cli/compose/project/project.go

```diff
@@ -34,6 +35,7 @@ type Project interface {
 
 	Context() *context.ECSContext
 	ServiceConfigs() *config.ServiceConfigs
+	ContainerConfigs() []containerconfig.ContainerConfig
```

Comment:
I may be misremembering, but did we decide to replace `ServiceConfigs` with `ContainerConfigs`, or are we just planning to do that step later?

Reply:
we did decide to replace ServiceConfigs, but plan to do that later. Would like to build the ContainerConfigs parsing/workflow up along side the ServiceConfigs and then remove it once the v1/2 parsing is working.